### PR TITLE
Documented /healthy, /ready API for alertmanager

### PR DESCRIPTION
**Pull request by MIBc**
Add document for alertnamager about some management api including the following:
/-/healthy
/-/ready
/-/reload

relevant issue: https://github.com/prometheus/alertmanager/issues/2039

### CODE REVIEW

**Comment by MIBc** [comment]
@simonpasquier Can you review it please ?

File: content/docs/alerting/management_api.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+### Reload
+
+```
+POST /-/reload
+```
+
+This endpoint triggers a reload of the Alertmanager configuration file. 
+
+An alternative way trigger a configuration reload is by sending a `SIGHUP` to the Alertmanager process.
```
Comment (simonpasquier) [inline content/docs/alerting/management_api.md]: "way to trigger"

Comment (MIBc) [reply content/docs/alerting/management_api.md]: > "way to trigger"

done

**Comment by MIBc** [comment]
@simonpasquier please review it

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**